ssv/spectest/tests/runner/postconsensus/unordered_expected_roots.go

```diff
@@ -25,7 +25,7 @@ func UnorderedExpectedRoots() tests.SpecTest {
 				),
 				Duty: testingutils.TestingAttesterDuty,
 				Messages: []*types.SignedSSVMessage{
-					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(nil, testingutils.PostConsensusAttestationMsg(ks.Shares[1], 1, testingutils.TestingDutySlot))),
+					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(ks, nil, testingutils.PostConsensusAttestationMsg(ks.Shares[1], 1, testingutils.TestingDutySlot))),
 				},
```

Comment:
how are the roots here unordered?

Reply:
Do we have a designated order here? I thought this was just for the sync committee contribution.